projects/crates.io/exa/package.yml

```diff
@@ -0,0 +1,25 @@
+distributable:
+  url: https://github.com/ogham/exa/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+provides:
+  - bin/exa
+
+versions:
+  github: ogham/exa/tags
+  strip: /^v/
+
+dependencies:
+  darwin:
+    zlib.net: ^1
```

Comment:
Is this really only required on mac? If so could use a comment explaining it.

Reply:
It seems to be required, without `zlib` the CI fails. After further investigation, it seems that I also need `pandoc` at build time for additional functionality, as well as `libgit2` on linux. I'll take a closer look